docs/user/ppl/cmd/fillnull.rst

```diff
@@ -0,0 +1,67 @@
+=============
+fillnull
```

Comment:
Need to update category.json in doctest to get this to run with the documentation test.

Reply:
Updated.